content/docs/csidriver/installation/helm/isilon.md

```diff
@@ -134,6 +134,8 @@ CRDs should be configured during replication prepare stage with repctl as descri
 
    | Parameter | Description | Required | Default |
    | --------- | ----------- | -------- |-------- |  
+   | version | Set to verify the values file version matches driver version and used to pull the image as part of the image name. | Yes | 2.5.0 |
```

Comment:
We ask the user not to change this param. We have already added a note in the values.yaml file. Please remove this version from here from the list of editable params.